unity-renderer/Assets/Scripts/MainScripts/DCL/Bridges/RealmsInfoBridge/RealmsInfoHandler.cs

```diff
@@ -21,7 +21,7 @@ public void Set(RealmsInfoModel newModel)
         {
             model = newModel;
             DataStore.i.realm.playerRealm.Set(model.current?.Clone());
-            DataStore.i.realm.realmsInfo.Set(model.realms);
+            DataStore.i.realm.realmsInfo.Set(model.realms, true);
```

Comment:
Why are we forcing the notification?

Reply:
I was forcing it because always that I received an update in the variable `DataStore.i.realm.realmsInfo` with the same number of items, the `Equals` method returned me a `true` (even if the data inside the `RealmModel` was different). Now I have discovered why it was happening... we had the variable declared as `BaseVariable<RealmModel[]>`. I have change it by a `BaseCollection<RealmModel>` and it works perfectly now. With this I no longer need to force the notification.

Thanks for the warning!